handUtils.js

```diff
@@ -11,65 +11,56 @@ export function checkStraightOrDrawOfLength(allCards, lengthOfDraw) {
 
   for (let i = 0; i <= uniqueRanksInOrder.length - lengthOfDraw; i++) {
     const subStringToCheck = uniqueRanksInOrder.slice(i, i + lengthOfDraw);
-    if ('AKQJT98765432A'.indexOf(subStringToCheck) !== -1) {
+    if ('AKQJT98765432A'.includes(subStringToCheck)) {
```

Comment:
I bet this will be faster if we switch to iterating through the unique ranks in order and see how many consecutive ranks we can find - not sure if that's worth it, but just a thought